cms/djangoapps/contentstore/views/course.py

```diff
@@ -1240,9 +1242,17 @@ def settings_handler(request, course_key_string):  # lint-amnesty, pylint: disab
                         set_prerequisite_courses(course_key, prerequisite_course_keys)
                     else:
                         # None is chosen, so remove the course prerequisites
-                        course_milestones = milestones_api.get_course_milestones(course_key=course_key, relationship="requires")  # lint-amnesty, pylint: disable=line-too-long
+                        course_milestones = milestones_api.get_course_milestones(
+                            course_key=course_key,
+                            relationship="requires",
+                        )
                         for milestone in course_milestones:
-                            remove_prerequisite_course(course_key, milestone)
+                            ee_milestone_namespace = generate_milestone_namespace(
```

Comment:
Agreed. It'd more clear if you use entrance_exam_namespace or something similar. Also, can you add a substantial comment explaining why you're adding this change? In case there's a similar issue in the future

Reply:
And that makes me wonder, do you think there are other milestones that this piece of code could remove? or just entrance exam?